momento-functions/examples/turbopuffer-recommended-articles.rs

```diff
@@ -149,7 +148,7 @@ fn get_recommended_articles(Json(request): Json<Request>) -> WebResult<WebRespon
 
 /// Gets existing article vector embeddings from our momento cache first,
 /// then queries (and caches) any misses from Turbopuffer itself
-fn get_cached_article_embeddings(
+fn get_article_embeddings(
```

Comment:
Since the embeddings will be out of order, and there's the possibility an embedding may not exist for an id (if the user passes in a bogus article id), can you have this function return a list of tuples: `(article_id, Option<corresponding_embedding>)`? The calling function can just gather all non-`None` embeddings and do the thing, but this should make it easier to debug.

Reply:
https://github.com/momentohq/functions/pull/6/commits/55505d86d84887e7a20dcc9c7ebfc3f443da2ead